issue-857-added a disclaimer for the gift registry inactivation info (#876)
magento/merchdocs: src/marketing/gift-registry-information.md

|Gift Registry Details|The Gift Registry Details might include additional information that is specific to the occasion.|
|Registrant Information|The Registrant Information section includes the name and contact information of each person who is to receive notification of the  registry. For a wedding registry, the Role field might be included to associate the registrant as a friend of the bride or groom.|
|Shipping Address|The Shipping Address section shows where gifts are to be sent, and includes the information a carrier needs to deliver the package.|

{:.bs-callout-info}

> When a gift registry is inactivated, the registry can't be searched or linked to. Also, if the registry is reactivated, links remain broken.

Reviewer: Edit for grammar and clarity

```suggestion
When a gift registry is inactivated, searching and linking does not function for the registry. If the registry is later reactivated, links remain broken.
```
